spec/features/admin_can_edit_an_item_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'admin' do
+  context 'with valid credentials' do
+    let(:admin){
+      User.create(first_name: "Admin",
+                  last_name: "astrator",
+                  username: "admin",
+                  email: "admin@admin.com",
+                  password: "password",
+                  password_confirmation: "password",
+                  role: 1)
+    }
+
+    let(:item){
+      Item.create(title: "Charizard",
+                  description: "Fire Breathing bastard",
+                  price: 2000)
+    }
+  it 'can edit an item' do
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.